agent/agent.go

```diff
@@ -1485,15 +1485,30 @@ func writeFileAtomic(path string, contents []byte) error {
 		return err
 	}
 	if _, err := fh.Write(contents); err != nil {
+		logger.Printf("[INFO] Writing to temp file at %v failed, deleting...\n", tempPath)
```

Comment:
maybe this was a different logging library but I seem to remember seeing it continue to log in the previous line. I'll double check. 

I liked keeping this info because it tells the user exactly what file is being deleted because of the error. (The error bubbled up states that saving the temp file failed, but that's not the same information as us saying that we deleted that afterwards). Can I make it debug instead?